### PR TITLE
nav/header ui iterate

### PR DESCRIPTION
**Pull request by brendansudol**
Iterates on nav design/layout per https://github.com/18F/cms-hitech-apd/issues/188

Adds icon library (font awesome) and using explicit importing (https://github.com/FortAwesome/react-fontawesome#usage) to keep our bundle size down.

Rejiggers location of `TopNav` so that sidebar span entire height (per mockup)

Hides sidebar (on mobile view) until we figure out how we want to handle that. (can still navigate via back and continue buttons)

Preview:
header

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- ~The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~
- ~The change has been documented~
  - ~Associated OpenAPI documentation has been updated~

### This feature is done when...
- [ ] Design has approved the experience

### CODE REVIEW

**Comment by brendansudol** [comment]
For consistency with where the "log in" button/link is, I kept the "log out" link in the same place (vs. at the bottom of sidebar) -- how do you feel about that?

**Comment by mgwalker** [comment]
Pinging @lauraponce!  😄

**Comment by lauraponce** [comment]
I feel good about that, looks good to me! 👍

File: web/src/containers/TopNav.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 
+// to optimize bundle, explicitly importing only the icons used
+// TODO: once we're using icons more, move these to a central location
```
Comment (mgwalker) [inline web/src/containers/TopNav.js]: We should probably add the Uglify plugin to WebPack to do tree-shaking, too...

File: web/src/containers/TopNav.js [hunk 2]
```diff
@@ -24,15 +31,25 @@ class TopNav extends Component {
...
-              <button type="button" className="btn" onClick={this.handleLogout}>
-                Log out
+              <button type="button" className="btn h5 regular">
+                <span className="mr-tiny">Notifications</span>
+                <FontAwesomeIcon icon={faBell} />
+              </button>
+              <button type="button" className="btn h5 regular">
+                <span className="mr-tiny">Help</span>
+                <FontAwesomeIcon icon={faHelp} />
+              </button>
```
Comment (mgwalker) [inline web/src/containers/TopNav.js]: I'm curious if the help button should really only be accessible if you're logged in.  Maybe a question for @lauraponce?  If it should always be available, is it okay to create an issue to fix it so we can go ahead and merge this?

File: web/src/styles/app/spacing.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/*
+  we need a spacing unit that's smaller than mb1, mt1, etc.
+  could call it "tiny" or "05"; using the former for now
+*/
+
+.mr-tiny {
```
Comment (mgwalker) [inline web/src/styles/app/spacing.css]: Mr. Tiny, the smallest man on his block.

File: web/package.json [hunk 4]
```diff
@@ -36,6 +36,10 @@
   },
   "homepage": "https://github.com/18F/cms-hitech-apd#readme",
   "dependencies": {
+    "@fortawesome/fontawesome": "^1.1.4",
```
Comment (mgwalker) [inline web/package.json]: I was a little worried about what FA does to a11y, but it turns out they're handling that internally, and this React wrapper doesn't interfere with it!  :tada:  Checked it locally to make sure it was rendering properly and everything looks gravy.

Comment (brendansudol) [reply web/package.json]: i thought they might, given its popularity

**Pull request merged**